### PR TITLE
SERV-442 Added teasers for Enterprise-only features

### PR DESCRIPTION
**Pull request by OndroMih**
Added teasers for Enterprise-only features. Also removed docs for Notifiers and Yubikey that were removed from the Community edition.

### CODE REVIEW

File: docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc [hunk 1]
```diff
@@ -5,22 +5,11 @@ The notification service comes with an integrated set of notifiers that
...
  and observing them in instances in the cluster.
-*Hipchat*:: You can configure the notification service to send events
-to an specific Hipchat chat room.
-*Slack*:: Same as with the Hipchat notifier, you can sends
-notifications directly to an specific Slack channel.
-*SNMP*:: Allows monitoring servers to scout for notification events
-using SNMP traps.
-*XMPP*:: Allows sending notification events to XMPP servers.
-*NewRelic*:: Allows sending notification events to a *NewRelic* service account.
-*Datadog*:: Allows sending notification events to the *Datadog* monitoring service
```
Comment (rdebusscher) [inline docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc:23]: Should we insert a link to the enterprise documentation indicating it has more notifiers available out of the box?

**Review by rdebusscher — APPROVED** [review]
A few optional comments

File: docs/modules/ROOT/pages/README.adoc [hunk 2]
```diff
@@ -6,6 +6,45 @@ This is the documentation for the Payara Platform Community Edition, targeted fo
 
 Payara Platform Community Edition provides a frequent release cycle to drive faster innovation. Payara Server Community and Payara Micro Community are dedicated to rapid change and innovation, giving developers a place to play with the latest features of the Payara Platform as soon as possible.
 
+== Payara Platform Editions
+
+The Payara Platform is available in 2 editions:
+
+* Payara Platform Community Edition - designed for development projects, or when innovation is higher priority than security and stability
```
[the remark this answers is not in the document]
Comment (OndroMih) [reply docs/modules/ROOT/pages/README.adoc:13]: I copied this from https://www.payara.fish/enterprise/enterprise-vs-community/ so I'd leave it as is.

File: docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc (hunk 1 above)
Comment (OndroMih) [reply docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc:23]: Sure, I didn't think about it. I'll do it.

**Pull request merged**